docs/guides/server/reverseproxy.adoc

```diff
@@ -233,6 +233,7 @@ to load additional certificates from headers `CERT_CHAIN_0` to `CERT_CHAIN_9` if
 | Whether the forwarded certificate is url-encoded or not. In NGINX, this corresponds to the `$ssl_client_cert` and `$ssl_client_escaped_cert` variables. This can also be used for the Traefik PassTlsClientCert middleware, as it sends the client certficate unencoded.
 |===
 
+NOTE: look into <@links.server id="configuration-provider"/>. For example you can use this parameter <@kc.start parameters="--spi-x509cert-lookup-nginx-ssl-client-cert=SSL_CLIENT_CERT"/>
```

Comment:
@igieon Thank you for the PR! Unfortunately, the rendering looks a bit off:
<img width="1100" height="175" alt="Image" src="https://github.com/user-attachments/assets/5cc65c8e-48a8-4d2d-a631-80e92493e21b" />

Maybe we could do something like this instead:
```suggestion
[NOTE]
====
Consult the <@links.server id="configuration-provider"/> guide for details on the format of the options. For example you can use:

<@kc.start parameters="--spi-x509cert-lookup-nginx-ssl-client-cert=SSL_CLIENT_CERT"/>
====

```
Which then looks like:
<img width="1100" height="137" alt="Image" src="https://github.com/user-attachments/assets/82ccfefb-8dce-4125-9b66-e2d2c9da205e" />